packages/jest-runtime/src/transform.js

```diff
@@ -330,9 +331,14 @@ const transformAndBuildScript = (
 module.exports = (
   filename: Path,
   config: Config,
-  options: ?Options,
+  options: Options = {},
```

Comment:
I usually keep this optional and do `options && options.instrument` to avoid the object allocation. I never benchmarked it though, so pretty sure engines optimize for your case too.

HOWEVER, some versions of node 4 don't support default args, so you either have to enable the babel transform for this or not use default args.